fjord/alerts/api_views.py

```diff
@@ -74,10 +28,9 @@ def is_after(value1, value2):
 
 
 class AlertsGETSerializer(StrictArgumentsMixin, serializers.Serializer):
+    """Serializer that validates GET API arguments"""
     flavors = serializers.CharField(required=True)
-    max = serializers.IntegerField(
-        required=False, default=100,
-        validators=[positive_integer])
+    max = serializers.IntegerField(default=100, min_value=1)
```

Comment:
Yeah! I figured `min_value=1` is less code than all the `positive_integer` stuff.